### DISCUSSION TITLE
Render in nuxt 3

### DISCUSSION THREAD

**Discussion by hnviradiya** [nuxt/nuxt#16568] [Questions]
I have built nuxt app using `buildNuxt` of nuxt kit.

Now how do we render it in expressJs. As in nuxt 2, we were able to do `nuxt.render(req, res)` where `nuxt` is instance of `import { Nuxt } from 'nuxt';`

**Comment by hnviradiya** [comment]
Following is the previous discussion, but seems I am unable to get it done without running it in separate process in the node.js

https://github.com/nuxt/framework/discussions/3650#discussioncomment-2372173

**Comment by danielroe** [comment]
This is possible to do but not recommended. You would likely want to implement with a different nitro preset. You could use the [`node` preset](https://github.com/unjs/nitro/blob/main/src/runtime/entries/node.ts#L5) for example and then run:

```js
import { handler } from './.output/server/index.mjs'

// your nest code
```

Alternatively you could write your own preset + entry that exactly does what you want.
